collectors/python.d.plugin/varnish/metadata.yaml

```
@@ -36,14 +44,48 @@ setup:
     list: []
```

```suggestion
    list:
      - title: Provide the necessary permissions
        description: |
          In order for the collector to work, you need to add the `netdata` user to the `varnish` user group, so that it can execute the `varnishstat` tool:
          
          ```
          usermod -aG varnish netdata
          ```
```